crt_portal/cts_forms/models.py

```diff
@@ -317,6 +318,12 @@ def get_absolute_url(self):
     def closed(self):
         return self.status == CLOSED_STATUS
 
+    def primary_complaint_description(self):
```

Comment:
Django has a built-in method we can leverage to provide this functionality since we're specifying a `choices` attribute on the model field! 

https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.get_FOO_display

`get_primary_complaint_display` will return the human readable value of `primary_complaint`